packages/@aws-cdk/integ-tests/test/integ.app.ts

```diff
@@ -0,0 +1,7 @@
+import { App, Stack } from '@aws-cdk/core';
```

Comment:
This isn't testing anything. Why is this here?

Reply:
@TheRealAmazonKendra, I can actually remove this I added it to avoid the linter issue saying PR must have at least one integration test update. But integ-runner doesn't have its own integration tests that's why I added this small test. And with this test `integ-tests/package.json` will also be reverted.